docs/data/charts/bars/bars.md

```diff
@@ -61,3 +61,21 @@ Bar charts can be rendered with a horizontal layout by providing the `layout="ho
 If you're using [composition](/x/react-charts/#multiple-charts), you should set the property `layout: 'horizontal'` to each bar series object.
 
 {{"demo": "HorizontalBars.js"}}
+
+## Animation
+
+To animate the creation and update of your chart use the `animate` prop.
```

Comment:
Have you considered making the `animation` opt-out instead of `opt-in`? 🤔 
It looks like most libraries do have animation toggled on by default. 🤷

Reply:
The reason is that I'm not fully sure about the solution. I prefer to have it `opt-in` such that users willing to get animation can easily add it, and report them if there is some. Other who don't care don't get bothered with them.

Until the end of v7 beta it will let more than 3 months to battle test those animation and them move them `opt-out`

Reply:
> The reason is that I'm not fully sure about the solution.

Then my suggestion is even stronger, I'd definitely go with `opt-out` to increase the time to problem discovery. 🤔 
I think that very few users might want to use charts without animation, although, I might be mistaken. 🙈

Reply:
18k people disagree with you 😁

But that's a good point 🤔

Reply:
We don't know how many of those downloads end up converting into usages in production environments. 🤔 
Sure, for admin panels, it's no concern, I was referring more toward regular user-facing applications. 🙈